indigo/distribution.yaml

```diff
@@ -10806,6 +10806,23 @@ repositories:
       url: https://github.com/rst-tu-dortmund/teb_local_planner.git
       version: master
     status: developed
+  teleop-twist-joy-xbot-release:
+    doc:
+      type: git
+      url: https://github.com/giupal/teleop_twist_joy_xbot-release.git
+      version: 0.1.2
+    release:
+      packages:
+      - teleop_twist_joy_xbot
```

Comment:
Please remove the `packages` entry once the only package name matches the repo name.